hikari/impl/stateful_cache.py

```diff
@@ -670,8 +703,7 @@ def __init__(self, app: traits.RESTAware, intents: typing.Optional[intents_.Inte
         self._invite_entries: typing.MutableMapping[str, _InviteData] = {}
         self._role_entries: typing.MutableMapping[snowflake.Snowflake, guilds.Role] = {}
         self._unknown_emoji_entries: typing.MutableMapping[
-            typing.Union[str, snowflake.Snowflake],
```

Comment:
what makes emojis unknown? Comments for context?

Reply:
technically that typing in the Generic wrapper should be an invariant typevar but it's kind of hard to explain what an unknown custom emoji with our current naming scheme